[validate] - fixed an issue with validate -g (#3573)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Fixed an issue where logging arguments were not in the standard kebab-case. The new arguments are: **console-log-threshold**, **file-log-threshold**, **log-file-path**.
   Added a validation to ensure there's a '.' in the end of each description field in integration and script yml files.

> * Fixed an issue where the **validate -g** failed when updating an empty `.pack-ignore` file when trying to retrieve its content from the local default branch.

Reviewer: ```suggestion
* Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that was previously empty.
```
